File: library/src/main/java/org/xmtp/android/library/GroupChat.kt
package uniffi.xmtp_dh.org.xmtp.android.library

import org.xmtp.android.library.Client
import uniffi.xmtp_dh.org.xmtp.android.library.codecs.GroupChatMemberAddedCodec
import uniffi.xmtp_dh.org.xmtp.android.library.codecs.GroupChatTitleChangedCodec

class GroupChat {
    companion object {
        fun registerCodecs() {
            Client.register(codec = GroupChatMemberAddedCodec())
            Client.register(codec = GroupChatTitleChangedCodec())
        }
    }
}

File: Sources/XMTP/GroupChat.swift
//
//  GroupChat.swift
//
//
//  Created by Pat Nakajima on 6/11/23.
//

import Foundation

public struct GroupChat {
	public static func registerCodecs() {
		Client.register(codec: GroupChatMemberAddedCodec())
		Client.register(codec: GroupChatTitleChangedCodec())
	}
}
